### PR TITLE
W-14749816-cli-mule46-release-cadence

### PR DESCRIPTION
**Pull request by luanamulesoft**

### CODE REVIEW

File: modules/ROOT/pages/_partials/cloudhub-apps.adoc [hunk 1]
```diff
@@ -183,6 +185,8 @@ Character `:` is not supported for the property's name.
 |--help                                                  |
 Output usage information
 |--timeout | Set the timeout value in miliseconds. Can take values between `60000` and `300000`.
+|--releaseChannel| Set the name of the release channel to be used for the selected Mule version. Supported values are `NONE`, `EDGE`, `LTS`. If you don't specify a Mule version, the default Mule version for the selected release channel is used. If the selected release channel doesn't exist, you get an error.
```
Comment (dmerlob) [inline modules/ROOT/pages/_partials/cloudhub-apps.adoc]:
```suggestion
|--releaseChannel| Set the name of the release channel to be used for the selected Mule version. Supported values are `NONE`, `EDGE`, and `LTS`. If you don't specify a Mule version, the default Mule version for the selected release channel is used. If the selected release channel doesn't exist, you get an error.
```

File: modules/ROOT/pages/_partials/cloudhub-apps.adoc [hunk 2]
```diff
@@ -183,6 +185,8 @@ Character `:` is not supported for the property's name.
...
+|--javaVersion | Set the name of the JAVA version to be used for the selected Mule version. Supported values are: `8` and `17`. If you don't specify a Mule version, the default Mule version for the selected JAVA version is used. If the JAVA version you select is not available for the specified Mule version, you get an error.
```
Comment (dmerlob) [inline modules/ROOT/pages/_partials/cloudhub-apps.adoc]:
```suggestion
|--javaVersion | Set the name of the JAVA version to be used for the selected Mule version. Supported values are `8` and `17`. If you don't specify a Mule version, the default Mule version for the selected JAVA version is used. If the JAVA version you select is not available for the specified Mule version, you get an error.
```

**Review by dmerlob — DISMISSED** [review]

**Review by dmerlob — DISMISSED** [review]

File: modules/ROOT/pages/_partials/cloudhub-apps.adoc [hunk 3]
```diff
@@ -185,6 +185,8 @@ Character `:` is not supported for the property's name.
 |--help                                                  |
 Output usage information
 |--timeout | Set the timeout value in miliseconds. Can take values between `60000` and `300000`.
+|--releaseChannel| Set the name of the release channel to be used for the selected Mule version. Supported values are `NONE`, `EDGE`, and `LTS`. If you don't specify a Mule version, the default Mule version for the selected release channel is used. If the selected release channel doesn't exist, you get an error.
```
Comment (rithishapadmanabh) [inline modules/ROOT/pages/_partials/cloudhub-apps.adoc]: Lets call out the default releaseChannel if it is not specified. I believe it is Edge

File: modules/ROOT/pages/_partials/cloudhub-apps.adoc [hunk 4]
```diff
@@ -185,6 +185,8 @@ Character `:` is not supported for the property's name.
...
+|--javaVersion | Set the name of the JAVA version to be used for the selected Mule version. Supported values are `8` and `17`. If you don't specify a Mule version, the default Mule version for the selected JAVA version is used. If the JAVA version you select is not available for the specified Mule version, you get an error.
```
Comment (rithishapadmanabh) [inline modules/ROOT/pages/_partials/cloudhub-apps.adoc]: Lets call out default Java version if it is not specified. It is 8 for Mule Runtime 4.6

File: modules/ROOT/pages/_partials/cloudhub-apps.adoc (hunk 3 above)
Comment (rithishapadmanabh) [reply modules/ROOT/pages/_partials/cloudhub-apps.adoc]: Might also be useful to call out default mule version for new app deploys is the latest mule version.

**Review by rithishapadmanabh — DISMISSED** [review]

File: modules/ROOT/pages/_partials/cloudhub-apps.adoc (hunk 3 above)
Comment (luanamulesoft) [reply modules/ROOT/pages/_partials/cloudhub-apps.adoc]: _> Lets call out the default releaseChannel if it is not specified. I believe it is Edge_
I asked the devs about it, and they confirmed that CLI does not impose any default value, but the CloudHub API does.



_> Might also be useful to call out default mule version for new app deploys is the latest mule version._
I asked the devs about it,  and they confirmed this clarification should be added to the --runtime flag.

File: modules/ROOT/pages/_partials/cloudhub-apps.adoc (hunk 4 above)
Comment (luanamulesoft) [reply modules/ROOT/pages/_partials/cloudhub-apps.adoc]: I asked the devs about it, and they confirmed that CLI does not impose any default value, but the CloudHub API does.

File: modules/ROOT/pages/_partials/cloudhub-apps.adoc [hunk 5]
```diff
@@ -159,7 +159,7 @@ The flags this command can take are:
 |===
 |Flag |Description
 |--runtime                                   | Name and version of the runtime environment. +
-Use this flag to specify the name and version of the runtime you want to deploy. +
+Use this flag to specify the name and version of the runtime you want to deploy. If you don’t specify a runtime version, CloudHub API deploys the latest version available considering the values you select for `--javaVersion` and `--releaseChannel` +
 Some examples of this value are `2.1.1-API-Gateway`, `3.9.1-visualizer` or   `4.1.1`. +
```
Comment (matbruc) [inline modules/ROOT/pages/_partials/cloudhub-apps.adoc]: @luanamulesoft please add to these examples `4.5` and `4.6-e-java8`

Comment (matbruc) [reply modules/ROOT/pages/_partials/cloudhub-apps.adoc]: `4.6-java8` , `4.6-e-java17`

Comment (matbruc) [reply modules/ROOT/pages/_partials/cloudhub-apps.adoc]: Remove the Note that reads:

```Starting with Mule 4.5, MuleSoft introduces two new release channels, Edge and Long-term Support (LTS). The Mule runtime versioning schema for the new release channels is:

Major[numeric] . Minor[numeric] . Patch[numeric] : Build[numeric] Channel[e for edge, nothing for LTS]

Some examples of these values are: 4.5.0:1e for Edge, and 4.6.0:1 for LTS.

For more information, see [Edge and LTS Releases for Mule](https://docs.mulesoft.com/release-notes/mule-runtime/lts-edge-release-cadence).````

**Review by dmerlob — APPROVED** [review]

**Pull request merged**